src/Game/UI/uiPauseMenuDataMgr.h

```diff
@@ -479,12 +479,12 @@ class PauseMenuDataMgr {
     mutable sead::CriticalSection mCritSection;
     Lists mItemLists;
     sead::SafeArray<PouchItem**, NumPouchCategories> mListHeads;
-    sead::SafeArray<PouchItem*, NumPouch50> mArray1;
-    sead::SafeArray<PouchItemType, NumPouch50> mArray2;
+    sead::SafeArray<PouchItem*, NumTabMax> mArrayTabs;
+    sead::SafeArray<PouchItemType, NumTabMax> mArrayTabsType;
     PouchItem* mLastAddedItem{};
-    s32 _44490 = -1;
-    s32 _44494 = -1;
-    s32 _44498{};
+    u32 mLastAddedItemTab = 0xFFFFFFFF;
```

Comment:
why was this changed to an unsigned int?

Reply:
I have seen this line in updateInventoryInfo : " *(uint *)(param_1 + 0x44490) = uVar20 - 1; "
I made this change because uVar20 is the number of tab you have in your inv

Reply:
That line alone does not tell you if it's signed or unsigned - Ghidra is only making a guess (especially when you haven't provided it with the proper type for `param_1`) and chances are that the assembly corresponding to that single line is exactly the same whether mLastAddedItemTab is signed or unsigned.

IMO it's more likely for this to be signed, given the -1 value, the other signed ints right next to this variable and Nintendo EPD tending to use signed ints.